File: JavaServletSrc/TransactionLibrary.js
//---Start function: CurrentTransactionStatus
function transactionlibrary_CurrentTransactionStatus() {
  return "";
}
//---End function

//---Start function: IsTransactionOk
function transactionlibrary_IsTransactionOk() {
  return true;
}
//---End function

//---Start function: LastTransactionStatus
function transactionlibrary_LastTransactionStatus() {
  return "";
}
//---End function

//---Start function: WasTransactionOk
function transactionlibrary_WasTransactionOk() {
  return true;
}
//---End function

//---Start function: MakeTransactionLibrary
function MakeTransactionLibrary() {
  this.CurrentTransactionStatus=transactionlibrary_CurrentTransactionStatus
  this.IsTransactionOk=transactionlibrary_IsTransactionOk
  this.LastTransactionStatus=transactionlibrary_LastTransactionStatus
  this.WasTransactionOk=transactionlibrary_WasTransactionOk
}
//---End function

var TransactionLibrary = new MakeTransactionLibrary()

File: JavaServletSrc/TransactionLibrary.java
public class TransactionLibrary {

//---Start function: TRANSACTIONLIBRARY_LIBRARY_CODE
  CPContext m_Ctx;
  public static TransactionLibrary Make(CPContext ctx) {
    TransactionLibrary l_TransactionLibrary = new TransactionLibrary();
    l_TransactionLibrary.m_Ctx = ctx;
    return l_TransactionLibrary;
  }
//---End function

//---Start function: CurrentTransactionStatus
public String CurrentTransactionStatus() {
  return m_Ctx.GetSql().CurrentTransactionStatus();
}
//---End function

//---Start function: IsTransactionOk
public boolean IsTransactionOk() {
  return m_Ctx.GetSql().IsTransactionOk("FileLibrary.IsTransactionOk");
}
//---End function

//---Start function: LastTransactionStatus
public String LastTransactionStatus() {
  return m_Ctx.GetSql().LastTransactionStatus();
}
//---End function

//---Start function: WasTransactionOk
public boolean WasTransactionOk() {
  return CPSql.TransactionWasOk("TransactionLibrary.WasTransactionOk");
}
//---End function

//---Start attributes
//---End attributes
}
